### ISSUE TITLE
On main page enable project filtering based on keywords

### ISSUE THREAD

**Issue by arno974** [3liz/lizmap-web-client#3774]
Would you think that it would be interesting to be able to filter map projects on the main page by using keywords ? Those keywords could be specified in the Qgis Server project properties and then be available on the main page. Something like on [this website](https://www.irds-idf.fr/cartographies-interactives/).

![image](https://github.com/3liz/lizmap-web-client/assets/425243/a705571f-201a-4baa-9a9a-19938f124064)

**Comment by Gustry** [comment]
It's already done by text typing, not a list of buttons.
It's hidden, because you need to click on the "T" button to make it "#".
The QGIS keywords list is already used.

Linked to #3767

**Comment by arno974** [comment]
Yes, I should have say that it could already been done with the search functionality. But it was more related to display the list of keywords as tags instead of a search bar. For end users it would be (I think) sometimes easier.
